## 3- Mês no qual há mais adições de filmes na plataforma Netflix

In [1]:
# importamos a biblioteca pandas
import pandas as pd

In [8]:
# carregamos dados do Netflix e forçamos a conversão da coluna 'date_added' a ser do tipo datetime
netflix = pd.read_csv('netflix_titles.csv', parse_dates=['date_added'])
# filtramos apenas por filmes
netflix = netflix[netflix['type']=='Movie']
# resetamos os indices
netflix.reset_index(inplace=True, drop=True)
netflix.head(2)

show_id   type                             title  \
0      s1  Movie              Dick Johnson Is Dead   
1      s7  Movie  My Little Pony: A New Generation   

                        director  \
0                Kirsten Johnson   
1  Robert Cullen, José Luis Ucha   

                                                cast        country  \
0                                                NaN  United States   
1  Vanessa Hudgens, Kimiko Glenn, James Marsden, ...            NaN   

  date_added  release_year rating duration                 listed_in  \
0 2021-09-25          2020  PG-13   90 min             Documentaries   
1 2021-09-24          2021     PG   91 min  Children & Family Movies   

                                         description  
0  As her father nears the end of his life, filmm...  
1  Equestria's divided. But a bright-eyed hero be...

In [10]:
# filtramos valores vazios (None)
netflix['date_added'] = netflix['date_added'][netflix['date_added'].notnull()]
# capturamos apenas o mês da coluna DATE_ADDED e fazemos uma contagem
netflix['date_added'].dt.month.value_counts()

7     565
4     550
12    547
1     546
10    545
3     529
9     519
8     519
11    498
6     492
5     439
2     382
Name: date_added, dtype: int64

### Como pode-se observar na saída anterior, o número 7(julho) obteve a maior quantidade de filmes adicionados (565 ao todo) à plataforma Netflix.